src/app/components/snackbar/snackbar.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { AnnouncementService } from '../../services/announcement/announcement.service';
+
+@Component({
+  selector: 'app-snackbar',
+  templateUrl: './snackbar.component.html',
+  styleUrls: ['./snackbar.component.css']
+})
+export class SnackbarComponent implements OnInit {
+  public message = '';
+  public klass = 'snackbar';
```

Comment:
Should this be 'class' instead of 'klass'? Am I missing something here?

Reply:
class is reserved name, so either className or klass (older convention).